drivers/hmis/app/models/concerns/hmis/mark_client_as_dirty_behavior.rb

```diff
@@ -0,0 +1,31 @@
+###
+# Copyright 2016 - 2025 Green River Data Analysis, LLC
+#
+# License detail: https://github.com/greenriver/hmis-warehouse/blob/production/LICENSE.md
+###
+
+# frozen_string_literal: true
+
+module Hmis::MarkClientAsDirtyBehavior
+  extend ActiveSupport::Concern
+
+  included do
+    after_save :mark_destination_client_dirty
+  end
+
+  protected
+
+  def mark_destination_client_dirty
+    # find the destination client
```

Comment:
Should we do a `return unless Hmis::Ce.configuration.enabled?` to reduce performance impact for non-CE installations?